dotcom-rendering/src/web/lib/decideDesign.ts

```diff
@@ -4,10 +4,10 @@ import { ArticleDesign } from '@guardian/libs';
  * NOTE: Immersive Opinion pieces are not supported,
  * i.e. when `CommentDesign` and `ImmersiveDisplay` are used jointly.
  */
-export const decideDesign = ({
-	design,
-	display,
-}: Partial<CAPIFormat>): ArticleDesign => {
+export const decideDesign = (
+	design: CAPIDesign | undefined,
+	display: CAPIDisplay | undefined,
+): ArticleDesign => {
```

Comment:
```suggestion
export const decideDesign = ({
	design,
	display,
}:{
	design: CAPIDesign | undefined,
	display: CAPIDisplay | undefined,
}): ArticleDesign => {
```

This is more verbose but passing objects is nice because there's not much chance for confusion!